gc/base/standard/Scavenger.cpp

```diff
@@ -4606,7 +4610,7 @@ MM_Scavenger::internalGarbageCollect(MM_EnvironmentBase *envBase, MM_MemorySubSp
 	}
 
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
-	if (_extensions->concurrentScavenger && isBackOutFlagRaised()) {
+	if (IS_CONCURRENT_ENABLED && isBackOutFlagRaised()) {
```

Comment:
while here, try removing #ifdef

Reply:
```
/gc/base/standard/Scavenger.cpp:4610:80: error: ‘J9MMCONSTANT_IMPLICIT_GC_PERCOLATE_ABORTED_SCAVENGE’ was not declared in this scope
 bool result = percolateGarbageCollect(env, subSpace, NULL, ABORTED_SCAVENGE, J9MMCONSTANT_IMPLICIT_GC_PERCOLATE_ABORTED_SCAVENGE);
```

Reply:
`J9MMCONSTANT_IMPLICIT_GC_PERCOLATE_ABORTED_SCAVENGE` is defined under CS